gcn/notices/km3net/test/medal_ranking_alert.schema.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/test/medal_ranking_alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "KM3NeT/Gold-Silver-Bronze Alerts",
+  "description": "Candidate neutrino observation reported from the KM3NeT online analysis pipeline.",
+  "allOf": [
+    {
+      "$ref": "../../core/Reporter.schema.json",
+      "description": "Instrument (ARCAxxx/ORCAxxx) issuing the alert. Mission=KM3NeT; Messenger=Neutrino."
+    },
+    {
+      "$ref": "../../core/Alert.schema.json",
+      "description": "Tense role (test/current observation), type (initial/update/retraction) and datetime of the alert generation"
+    },
+    {
+      "$ref": "../../core/Event.schema.json",
+      "description": "Event ID (eg 1A. Nb increment for each new alert, letter increment for update/retraction of the nb). Event_Name (KM3-yymmddL) if appropriate. "
+    },
+    {
+      "$ref": "#/$defs/space_time_coord",
+      "description": "Most probable Localization (Time, Position, Error and Skymap if available)."
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json",
+      "description": "FAR and other statistic related to the event probability."
+    },
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json",
+      "description": "Description of the pipeline used to produce the alert."
+    }
+  ],
+  "properties": {
+    "$schema": true,
+    "packet_type": {
+      "type": "number",
+      "description": "packet_type provided by GCN as for the old GCN format, associatied to notice_type/topic Gold=, Silver=, Bronze=."
+    },
+    "description": { "type": "string", "description": "Alert description." },
+    "analysis_pipeline": {
+      "type": "string",
+      "description": "Analysis pipeline outputing the alert (eg exceptionnal_evt_arca, multiplets_orca, ...)."
+    },
+    "triggering_evts": {
+      "type": "array",
+      "items": { "$ref": "#/$defs/km3event" },
+      "description": "Array of KM3NeT event space-time coord, topology and energy info."
+    },
+    "known_sources": {
+      "type": ["array", "string"],
+      "items": {
+        "anyOf": [
+          { "$ref": "#/$defs/knownsrc" },
+          { "$ref": "#/$defs/mmcoincevent" }
+        ]
+      },
+      "description": "No search / No match / Array of known astrophysical sources present in the KM3NeT event RoI (not exhaustive) OR reference and position for a coincident time-variable source."
+    }
+  },
+  "required": [
```

Comment:
If you with you can skip "required" option, it will enforce all fields must be there.

Reply:
If your pipeline will produce all the fields for each notices, then fine.
One can read at schema browser (https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices), for what to expect, even we will soon create and announce mission page soon after your feedback.
eg: https://gcn.nasa.gov/missions/einstein-probe
It will put constraint on your pipeline, it's upto you.